src/app/home-page/recent-item-list/recent-item-list.component.ts

```diff
@@ -59,7 +60,7 @@ import { VarDirective } from '../../shared/utils/var.directive';
     fadeInOut,
   ],
   standalone: true,
-  imports: [VarDirective, NgClass, ListableObjectComponentLoaderComponent, ErrorComponent, ThemedLoadingComponent, AsyncPipe, TranslateModule],
+  imports: [VarDirective, NgClass, ListableObjectComponentLoaderComponent, ErrorComponent, ThemedLoadingComponent, AsyncPipe, TranslateModule, RSSComponent],
```

Comment:
Small thing...is this RSSComponent still needed in the recent-item-list?  It looks unused

Reply:
thanks, removed